modules/auxiliary/server/capture/ldap.rb

```diff
@@ -35,159 +32,44 @@ def initialize(info = {})
 
     register_options(
       [
-        OptAddress.new('SRVHOST', [ true, 'The localhost to listen on.', '0.0.0.0' ]),
-        OptPort.new('SRVPORT', [ true, 'The local port to listen on.', '389' ])
+        OptAddress.new('SRVHOST', [ true, 'The ip address to listen on.', '0.0.0.0' ]),
+        OptPort.new('SRVPORT', [ true, 'The port to listen on.', '389' ]),
+        OptString.new('CHALLENGE', [ true, 'The 8 byte challenge', Rex::Text.rand_text_alphanumeric(16) ])
       ]
     )
 
     deregister_options('LDIF_FILE')
 
     register_advanced_options(
       [
+        OptString.new('Domain', [ false, 'The default domain to use for NTLM authentication', 'DOMAIN']),
+        OptString.new('Server', [ false, 'The default server to use for NTLM authentication', 'SERVER']),
+        OptString.new('DnsName', [ false, 'The default DNS server name to use for NTLM authentication', 'SERVER']),
+        OptString.new('DnsDomain', [ false, 'The default DNS domain name to use for NTLM authentication', 'example.com']),
+        OptBool.new('ForceDefault', [ false, 'Force the default settings', false]),
```

Comment:
Just a note, this option is not `used` by the implementation. 

Looking at where it came from, I suspect this is from intention to mimic `http_nltm` behavior.  For now I will just remove this options during land of this PR.